scripts/calendar/mongolian_generator.py

```
@@ -0,0 +1,204 @@
+    return gregorian_date(jd)
+
+
+def get_tsagaan_sar(y):
+    try:
+        jd = julian_day(y - 1, 12, False, 30) + 1
+        return gregorian_date(jd)
+    except Exception:
+        return gregorian_date(julian_day(y - 1, 12, True, 30) + 1)
+
+
+def find_festival_date(y, m_month, m_day):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)